src/utils/tokens.js

```diff
@@ -0,0 +1,224 @@
+import { ReactComponent as eth } from '../../node_modules/cryptocurrency-icons/svg/icon/eth.svg'
+import { ReactComponent as usdc } from '../../node_modules/cryptocurrency-icons/svg/icon/usdc.svg'
+import { ReactComponent as usdt } from '../../node_modules/cryptocurrency-icons/svg/icon/usdt.svg'
+import { ReactComponent as dai } from '../../node_modules/cryptocurrency-icons/svg/icon/dai.svg'
+import { ReactComponent as wbtc } from '../../node_modules/cryptocurrency-icons/svg/icon/wbtc.svg'
+import { ReactComponent as uni } from '../../node_modules/cryptocurrency-icons/svg/icon/uni.svg'
+import { ReactComponent as sushi } from '../../node_modules/cryptocurrency-icons/svg/icon/sushi.svg'
+import { ReactComponent as comp } from '../../node_modules/cryptocurrency-icons/svg/icon/comp.svg'
+import { ReactComponent as bal } from '../../node_modules/cryptocurrency-icons/svg/icon/bal.svg'
+import { ReactComponent as aave } from '../../node_modules/cryptocurrency-icons/svg/icon/aave.svg'
+import { ReactComponent as yfi } from '../../node_modules/cryptocurrency-icons/svg/icon/yfi.svg'
+import { ReactComponent as link } from '../../node_modules/cryptocurrency-icons/svg/icon/link.svg'
+import { ReactComponent as snt } from '../../node_modules/cryptocurrency-icons/svg/icon/snt.svg'
+import { ReactComponent as bat } from '../../node_modules/cryptocurrency-icons/svg/icon/bat.svg'
+import { ReactComponent as matic } from '../../node_modules/cryptocurrency-icons/svg/icon/matic.svg'
+import { ReactComponent as mkr } from '../../node_modules/cryptocurrency-icons/svg/icon/mkr.svg'
+import { ReactComponent as bnt } from '../../node_modules/cryptocurrency-icons/svg/icon/bnt.svg'
+import { ReactComponent as grt } from '../../node_modules/cryptocurrency-icons/svg/icon/grt.svg'
+import { ReactComponent as zrx } from '../../node_modules/cryptocurrency-icons/svg/icon/zrx.svg'
+import { ReactComponent as gno } from '../../node_modules/cryptocurrency-icons/svg/icon/gno.svg'
+import { ReactComponent as gnt } from '../../node_modules/cryptocurrency-icons/svg/icon/gnt.svg'
+import { ReactComponent as ant } from '../../node_modules/cryptocurrency-icons/svg/icon/ant.svg'
+import { ReactComponent as crv } from '../../node_modules/cryptocurrency-icons/svg/icon/crv.svg'
+import { ReactComponent as generic } from '../../node_modules/cryptocurrency-icons/svg/icon/2give.svg'
+
+const tokens = {
```

Comment:
We need a more flexible mechanism. We can use the information from /accounts or /tokens and the do `toLowerCase` with the symbol attribute and get the correct icon based on that

Reply:
It's not going to be that simple by the moment due to tree shaking. Either you import all the library (175+ icons) or you select before hand which icons you want to import. It's just temporary until we find a better solution which probably is going to require more infrastructure (store all the svgs in a server and just request the logos by using the coin symbol) etc.

Reply:
Well, I didn't see all the code. I would keep just a `Map` of `id` as a key and `icon` as the value. We should use the rest of the information from the API (name, symbol etc)

Reply:
I've cleaned the object tokens and link the icon with the symbol in lowercase